data/styles/clear/include/Basemap.mapcss

```diff
@@ -299,6 +299,9 @@ area|z12-13[landuse=village_green],
 area|z12-13[landuse=field],
 {fill-color: @green0;}
 
+area|z14-[landuse=farmland],
+{fill-color: @farmland;fill-opacity: 1;}
```

Comment:
Remove useless entry here, you put it in separate category below.
And remove area|z14-[landuse=farmland], entry above in Ln 266